app/templates/build.gradle

```diff
@@ -3,8 +3,8 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath("org.springframework.boot:spring-boot-gradle-plugin:1.2.5.RELEASE")
-    classpath("io.spring.gradle:dependency-management-plugin:0.5.2.RELEASE")
+    classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.0.RELEASE")
```

Comment:
It's not :). Spring cloud downgrades it to 1.2.6 again for now. Only the latest Brixton (kinda nightly) version works with 1.3.